torch_harmonics/convolution.py

```diff
@@ -347,11 +349,11 @@ def get_psi(self):
         psi = torch.sparse_coo_tensor(self.psi_idx, self.psi_vals, size=(self.kernel_size, self.nlat_out, self.nlat_in * self.nlon_in)).coalesce()
         return psi
 
-    def forward(self, x: torch.Tensor, use_triton_kernel: bool = True) -> torch.Tensor:
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
         # pre-multiply x with the quadrature weights
         x = self.quad_weights * x
-        
-        if x.is_cuda and use_triton_kernel:
+
+        if x.is_cuda:
             #x = _disco_s2_contraction_triton(x, psi, self.nlon_out)
             x = disco_cuda.forward(x, self.psi_roff_idx, self.psi_ker_idx, self.psi_row_idx, self.psi_col_idx, self.psi_vals, self.kernel_size, self.nlat_out, self.nlon_out)
```

Comment:
this is weird, I thought I replaced this with _disco_s2_contraction_cuda(x, self.psi_roff_idx, self.psi_ker_idx, self.psi_row_idx, self.psi_col_idx, self.psi_vals,
                                           self.kernel_size, self.nlat_out, self.nlon_out)

can you check please?

Reply:
i reabsed the PR, please double-check.